compiler/ras/DebugCounter.cpp

```diff
@@ -304,10 +304,13 @@ void TR::DebugCounterAggregation::aggregateStandardCounters(TR::Compilation *com
 void TR::DebugCounterAggregation::aggregateDebugCounterInsertions(TR::Compilation *comp, TR::Node *node, TR::DebugCounter *counter, int32_t delta, int8_t fidelity, int32_t staticDelta)
    {
    const char *counterNames[3];
-   counter->getInsertionCounterNames(comp, node, counterNames);
 
-   if (counter && counter->getDenominator() && counter->contributesToDenominator())
-      aggregateDebugCounterInsertions(comp, node, counter->getDenominator(), delta, fidelity, staticDelta);
+   if (counter)
```

Comment:
Having a null counter makes no sense so a TR_ASSERT followed by an early return would make sense to me.